### DISCUSSION TITLE
Parabéns para vocês que constroem essa comunidade!

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#777] [Discussão]
De vários e vários projetos que idealizei e participei nos últimos anos, tenho um orgulho enorme dessa nossa organização ❤️ 

O próprio TechLead do Github, quando veio no Cubo, ficou surpreso por uma ecossistema inteiro ter sido criado dentro do Github. Ele disse que sabia que existiam comunidades aqui, mas todas crescendo em torno de libs (React, Angular...). Ele disse que não tinha noção que uma comunidade tinha nascido como uma comunidade por si só.

Aqui várias pessoas ensinam, aprendem e arrumam emprego. 
Daqui surgiram outras iniciativas sensacionais, como o @training-center (@woliveiras, @alinebastos...e todo mundo que está envolvido, vocês detonam)!
 
E esse ano nos tornamos uma comunidade oficial de um evento do porte do Intercon 2017 do iMasters​ e isso é sensacional.

Quero aproveitar para divulgar uma lista que o @eduardojmatos criou, [com as pessoas mais participativas daqui do Fórum](https://gist.github.com/eduardojmatos/c370304fcfc4527b6da98bbb8f620783), o número de participações é legal, mas sabe o mais foda? 

O número de pessoas que participaram! Muita gente postou, nem que seja uma vez, e tantas outras participam indiretamente, acompanhando as issues e divulgando os conteúdos.

Meus parabéns para todas as pessoas que participaram e ajudaram na construção disso. 

Vocês são fodas e merecem todas as palmas 👏 👏 👏 👏 ! 

![image](https://user-images.githubusercontent.com/3603793/31839909-e1cb3daa-b5c1-11e7-9570-1530eb5234f9.png)

**Comment by eduardojmatos** [comment]
Muito legal isso tudo que rola aqui no frontendbr. Não imaginávamos que ía chegar nesse ponto que estamos hoje! Sabendo que tem muita coisa pra fazer, muita coisa pra colaborar! 

Me sinto extremamente orgulhoso em fazer parte dessa comunidade!

**Comment by alinebastos** [comment]
Que comunidade linda :heart:

**Comment by willycamargo** [comment]
O que está acontecendo aqui é diferente de tudo que aconteceu em outras fóruns ou grupos que conheci desde 2012.

E todos aqueles que buscam estar ligados na comunidade com real interesse em ajudar sabem disso. Parabéns a todos os organizadores e também a comunidade que mantém isso acontecendo!

**Comment by willianjusten** [comment]
Muito lindo isso aí, tenho orgulho de fazer parte disso desde o primeiro tweet xD
E vamos que vamos!

**Comment by matheusml** [comment]
Ajudar a comunidade sempre que for possível.
Essa é minha vida.
Esse é meu clube.

Ou melhor, **nosso** clube.

**Comment by suissa** [comment]
@LFeh Caramba até eu tô na lista ehehhee.
Espero começar a contribuir mais aqui também apesar de já ter uma legião de alunos que ajudo constantemente em pvt e nos mneus artigos e aulas.

Galera aqui está de parabéns e o futuro será melhor ainda.

Congratz folks!

**Comment by galvaodev** [comment]
Comunidade linda <3

**Comment by kvnol** [comment]
Passei pra falar que você são bonitinhos e que TAMO JUNTO SEMPRE :heart:

**Comment by nandomoreirame** [comment]
Coisa linda, parabéns! 💟

**Comment by JoseCage** [comment]
Opa.. Valeu por mencionar.. Nem fazia ideia que já participei essas 22 vezes rsrs..

De Angola (:angola:)  para o mundo.. Viva programação :+1: :+1: 

E parabéns pela iniciativa..

Tenho tido a paciência de abrir uma por uma `issue` (assinei para receber notificações) para ler. E para ser sincero, aprendo muito..

:+1: :brazil:

**Comment by leonantvrs** [comment]
Parabéns para comunidade, para todos que ajudam compartilhando conhecimento, tirando as dúvidas ou fazendo alguma coisa...

Pretendo ajudar mais no que for possível.

:smile:

**Comment by felquis** [comment]
🇧🇷 Go Brazil 🇧🇷
❤🇧🇷 777 🇧🇷❤

**Comment by WandersonAlves** [comment]
Que FODASTICO

Encontrei esse lugar numa pesquisa do Google e nunca mais sai.

Depois me vi maravilhado no grupo da vuejsbr no telegram e vi que sim: é possível unir pessoas, interagir com a PAIXÃO de desenvolver.

Muito obrigado por nos permitir esse momento <3

(E caramba, participei 48 vezes, se tivesse o quanto SÓ leio aqui iroeowha)

**Comment by dsoldera** [comment]
Apaixonada por essa Comunidade ❤️ ❤️ ❤️

**Comment by weltonsanches** [comment]
Essa foi a primeira comunidade que participei e não tenho palavras pra dizer o quanto tenho aprendido. Vocês são demais galera! :heart:

**Comment by theandersonn** [comment]
:clap: :clap: :clap: :clap: :clap:

**Comment by fabriciofmsilva** [comment]
Show demais! O Brasil precisa de boas comunidades como o Front-EndBR! Parabéns a todos.

**Comment by woliveiras** [comment]
Eu tive sorte de encontrar a comunidade front-end no geral e esse forum tem sido um lugar fantástico pra compartilhar conhecimento!

Parabéns galera!

Vocês são muito feras!

**Comment by mateusmiguel** [comment]
Tenho aprendido muito com todos vocês desde que cheguei na comunidade. 

Parabéns galera! <3

**Comment by mahenrique94** [comment]
Uma comunidade de dar orgulho, show <3

Bora crescer ainda mais \o/

**Comment by caiofsouza** [comment]
Vi o stand la e fiquei bem feliz pela conquista hein cara, parabens <3

**Comment by felipefialho** [comment]
@caiofsouza Parabéns pra nós!

**Comment by munizart** [comment]
Parabéns aos envolvidos, muito legal ver tanta gente preocupada em criar um ambiente de troca e ajuda mutua!

PS: muito legal a lista do @eduardojmatos, achei que tinha participado bem menos por aqui

**Comment by yanmagale** [comment]
Muito <3

Parabéns mesmo a todos os envolvidos. Vocês estão fazendo um trabalho incrível.

**Comment by diessica** [comment]
gosto do nível das discussões aqui! fórum lindo ❤️

**Comment by danilovaz** [comment]
Só discussões ricas em conteúdo e respeito, sempre ❤️

**Comment by brunoravanhani** [comment]
Parabéns pra todos nós galera

**Comment by didiraja** [comment]
Parabéns a todos nós <3

**Comment by rogeralbinoi** [comment]
Parabéns galera!!! 🚀🚀🚀

**Comment by darthrodolfo** [comment]
Obrigado a quem contribuiu e deixou esse montão de conteúdo pra quem chegou agora aprender e utilizar tambem hehe!

**Comment by Fernando-Braulio** [comment]
Chegando agora e já estou amando a comunidade. 
Obrigado a quem contribuiu, tem muitos conteúdos excelentes.

**Comment by luciano-veiga** [comment]
Que orgulho ver uma comunidade construída dessa forma: compartilhando conhecimento, ajudando uns aos outros e criando oportunidades. ❤️

Mesmo chegando depois, é muito bom encontrar todo esse conteúdo e perceber o impacto que a comunidade continua gerando.

Parabéns a todos que contribuíram e ajudaram a construir essa história! 🚀
